app/Database/CourseQueries.hs

```diff
@@ -36,7 +37,51 @@ import Data.Int (Int64)
 import Database.DataType
 import Svg.Builder
 
--- ** Querying a single course
+-- | Queries db for all matching records with lecture or tutorial code of this course
+lectureQuery :: MonadIO m => T.Text -> ReaderT SqlBackend m a 
```

Comment:
Oh also the last part of the type, the `a`, should really be the type of value which results from the query. In this case it would be something like `[Entity Lecture]`.